src/plugins/nestedRows/test/nestedRows.e2e.js

```diff
@@ -104,6 +104,19 @@ describe('NestedRows', () => {
       expect(getData()).toEqual(dataInOrder);
     });
 
+    it('should not move rows when they are on the highest level of nesting (don\'t have a parent)', () => {
```

Comment:
I thought that this test checks if there is no error while parent moving https://github.com/handsontable/handsontable/pull/6935#issuecomment-637535697. However, after reverting the changes which block parents moving (and prevents errors) the test still passes.

I'd add a test for the specific case described in the above GitHub comment. There I removed all parent's children then I move the parent between top rows, then the error occurs.